pkg/app/piped/platformprovider/ecs/ecs.go

```diff
@@ -57,7 +57,9 @@ type ECS interface {
 
 type ELB interface {
 	GetListenerArns(ctx context.Context, targetGroup types.LoadBalancer) ([]string, error)
+	GetListenerRuleArns(ctx context.Context, listenerArns []string) ([]string, error)
 	ModifyListeners(ctx context.Context, listenerArns []string, routingTrafficCfg RoutingTrafficConfig) error
+	ModifyRules(ctx context.Context, listenerRuleArns []string, routingTrafficCfg RoutingTrafficConfig) error
```

Comment:
No implementation found currently, I guess have to rename `ModifyListenerOrRule` name to this

Reply:
Thank you,Correspondence
[3a4bea5](https://github.com/pipe-cd/pipecd/pull/4668/commits/3a4bea5ab8f0f8baf79ea9734b6a91563c9424e8)